Implemented `get_rotation_matrix` to compute the rotation matrix between two SPICE frames. (#7452)
sunpy/sunpy: changelog/7452.feature.rst

> Implemented the `~sunpy.coordinates.spice.get_rotation_matrix` which computes the rotation matrix that converts vectors from the source frame to the target frame

Reviewer: ```suggestion
Implemented the `~sunpy.coordinates.spice.get_rotation_matrix` which computes the rotation matrix that converts vectors from the source frame to the target frame
```
```suggestion
Implemented `sunpy.coordinates.spice.get_rotation_matrix` which computes the rotation matrix that converts vectors from the source frame to the target frame
```

> Implemented the `~sunpy.coordinates.spice.get_rotation_matrix` which computes the rotation matrix that converts vectors from the source frame to the target frame

Reviewer 2: ```suggestion
Added :func:`sunpy.coordinates.spice.get_rotation_matrix` to obtain the rotation matrix between the orientations of two SPICE frames, which is particularly useful for transforming vector fields.
```

> Added :func:`~sunpy.coordinates.spice.get_rotation_matrix` to obtain the rotation matrix between the orientations of two SPICE frames, which is particularly useful for transforming vector fields
> at the specified time.

Reviewer 2: ```suggestion
Added :func:`~sunpy.coordinates.spice.get_rotation_matrix` to obtain the rotation matrix between the orientations of two SPICE frames, which is particularly useful for transforming vector fields.
```
